### ISSUE TITLE
Use react-navigation-parser in web

### ISSUE THREAD

**Issue by agrcrobles** [Kureev/react-navigation-parser#2]
Hey @Kureev the project is great, you have set a good point here.

When react-navigation was released I played for a while doing [react-navigation-web](https://github.com/agrcrobles/react-navigation-web) as an experiment to see if possible develop missing built in views for web.

Now I find this library could be great to parse and identify everything related to the scene/card.

Do you have an starting point or suggestion on how should that be done?

**Comment by Kureev** [comment]
Hi @agrcrobles! Thanks for the warm words!

What exactly do you want to parse? Configuration for the routes etc?

**Comment by agrcrobles** [comment]
exactly, any information I can get on runtime of how routes are configurated on web ( ie, title or path ) I should say that is a great start

**Comment by Kureev** [comment]
This can currently not supported, *but* I'd like to implement it. If you want to make a PR, the best place to implement it is here https://github.com/Kureev/react-navigation-parser/blob/master/src/parser/resolveRoute.js. Otherwise, just let me know and I'll try to do my best implementing it asap
